pom.xml

```diff
@@ -1,78 +1,65 @@
-	</parent>
-	<groupId>com.plhal.ares</groupId>
-	<artifactId>ares-api</artifactId>
-	<version>${project.version}</version>
-	<name>ares-api</name>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+    <properties>
+        <java.version>11</java.version>
+    </properties>
+    <modules>
+        <module>webapp</module>
+        <module>parser</module>
+        <module>service</module>
+        <module>dbLayer</module>
+    </modules>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.plhal.ares</groupId>
+    <artifactId>ares-api</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>ares-api</name>
 
-	<dependencies>
-		<dependency>
-			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-engine</artifactId>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-api</artifactId>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-devtools</artifactId>
-			<scope>runtime</scope>
-			<optional>true</optional>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-			<dependency>
-				<groupId>org.projectlombok</groupId>
-				<artifactId>lombok</artifactId>
-				<version>1.18.16</version>
-				<scope>provided</scope>
-			</dependency>
-	</dependencies>
-
-	<build>
-		<plugins>
-			<plugin>
-				<artifactId>maven-surefire-plugin</artifactId>
-			</plugin>
-			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-				<configuration>
-					<skip>true</skip>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-devtools</artifactId>
+            <scope>runtime</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Co toto v hlavnim modulu definovat jako dependancyManagement a na primo zavislost pridavat v sub-modulech?

Reply:
Tohle predpokladam, ze myslite k te lombok dependency?

Reply:
nn, myslel jsem [depdendancyManagement v mavenu](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html).

Reply:
Myslíte přidat do dependencyManagement dependency jako spring-boot-starter-web atd. ? A tyto dependency potom přidávat v podmodulech? Když si dám zobrazit efective pom toho hlavního modulu, tak to už v dependencyManagementu mám (zdědilo to od parent - spring-boot-starter-parent). Takže stačí když to přidám pouze v pod-modulech. Nebo jste to myslel jinak?

Reply:
Ano, mate uplnou pravdu. Dobra pripominka. A ano, tak jsem to myslel. Proste jednoduse pouzit ty zavislosti tam, kde museji byt. Hlavni projekt je temr urcite nepotrebuje (napriklad lombok).